mlir/unittests/Target/LLVM/SerializeToLLVMBitcode.cpp

```diff
@@ -55,7 +55,7 @@ TEST_F(MLIRTargetLLVM, SKIP_WITHOUT_NATIVE(SerializeToLLVMBitcode)) {
   ASSERT_TRUE(!!module);
```

Comment:
@fabianmcg : why is the test decorated with `SKIP_WITHOUT_NATIVE` ?

(I can't comment above, it's line 41)